src/MainWindow.vala

```diff
@@ -186,11 +186,37 @@ public class AppCenter.MainWindow : Gtk.ApplicationWindow {
 
         spinner = new Gtk.Spinner ();
 
+#if POP_OS
+        var repos_button = new Gtk.Button.from_icon_name ("preferences-system-symbolic", Gtk.IconSize.SMALL_TOOLBAR);
+        repos_button.tooltip_text = _("Edit Software Sources…");
+        repos_button.valign = Gtk.Align.CENTER;
+        repos_button.clicked.connect (() => {
+            try {
+                string[] args = {
+                  "/usr/lib/repoman/repoman.pkexec"
```

Comment:
I'm not a fan of this particular change. Generally speaking, AppCenter is designed to provide a default experience. If we were going to have software sources managed in AppCenter, I'd rather it be done through PackageKit and Flatpak APIs, rather than subprocessing out to some other program.